test/unit/cute/CMakeLists.txt

```diff
@@ -63,15 +63,15 @@ else()
   
   add_subdirectory(layout)
   add_subdirectory(core)
+  add_subdirectory(msvc_compilation)
```

Comment:
@aacostadiaz , I had not added `msvc_compilation` on purpose, as we do not use the Microsoft visual C++ compiler,
is this still required ?